### PR TITLE
Handle delay seconds in send_sms lambda

### PR DESCRIPTION
**Pull request by jeffreysilver**

### CODE REVIEW

File: stopcovid/send_sms.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+def _send_message_to_twilio(record):
+    message = json.loads(record["body"])
+    sleep(_get_delay_seconds(record))
+    return twilio.send_message(message["To"], message["Body"])
+
+
+def send_sms(event, context):
+    twilio_responses = [_send_message_to_twilio(record) for record in event["Records"]]
+    kinesis.publish_log_outbound_sms(twilio_responses)
```
Comment (pdsarin) [inline stopcovid/send_sms.py:18]: note that, if this fails, we'll end up double sending. I'd probably log+ignore any error here.

File: stopcovid/send_sms.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+from stopcovid.clients import twilio, kinesis
+
+
+def _get_delay_seconds(record):
+    return int(record["messageAttributes"].get("delay_seconds", {}).get("stringValue", "0"))
+
+
+def _send_message_to_twilio(record):
+    message = json.loads(record["body"])
+    sleep(_get_delay_seconds(record))
```
Comment (pdsarin) [inline stopcovid/send_sms.py:12]: is sleep(0) a no-op?

Comment (jeffreysilver) [reply stopcovid/send_sms.py:12]: Yes

File: stopcovid/send_sms.py (hunk 1 above)
Comment (jeffreysilver) [reply stopcovid/send_sms.py:18]: Good catch. Ill wrap it in a try except

**Pull request merged**